src/index.ts

```diff
@@ -17,3 +20,7 @@ const server = new GraphQLServer({
   resolvers,
 })
 server.start(() => console.log(`Server is running on http://localhost:4000`))
+
+createConnection().then(async connection => {
+  console.log('Connected successfully');
```

Comment:
Esse tipo de informação é legal estar num `console.info`. Em alguns sistemas pode ter hierarquia de tipografia neles. Mesma coisa pro `Server is running on...`